apps/api/src/server/applications/search/__tests__/case-search.test.js

```diff
@@ -0,0 +1,172 @@
+import test from 'ava';
+import sinon from 'sinon';
+import supertest from 'supertest';
+import { app } from '../../../app.js';
+import { applicationFactoryForTests } from '../../../utils/application-factory-for-tests.js';
+import { databaseConnector } from '../../../utils/database-connector.js';
+
+const request = supertest(app);
+
+const application = applicationFactoryForTests({
+	id: 3,
+	status: 'open',
+	reference: 'EN010003',
+	title: 'EN010003 - NI Case 3 Name',
+	description: 'EN010003 - NI Case 3 Name Description',
+	createdAt: new Date(1_655_298_882_000),
+	modifiedAt: new Date(1_655_298_882_000)
+});
+
+const applicationsCount = 1;
+
+const findManyStub = sinon.stub();
+
+findManyStub
+	.withArgs({
+		skip: 0,
+		take: 1,
+		orderBy: [
+			{
+				createdAt: 'desc'
+			}
+		],
+		where: {
+			OR: [
+				{
+					title: { contains: 'EN010003 - NI Case 3 Name' }
+				},
+				{
+					reference: { contains: 'EN010003 - NI Case 3 Name' }
+				},
+				{
+					description: { contains: 'EN010003 - NI Case 3 Name' }
+				}
+			]
+		},
+		include: {
+			subSector: {
+				include: {
+					sector: true
+				}
+			}
+		}
+	})
+	.returns([application]);
+
+const countStub = sinon.stub();
+
+countStub
+	.withArgs({
+		where: {
+			OR: [
+				{
+					title: { contains: 'EN010003 - NI Case 3 Name' }
+				},
+				{
+					reference: { contains: 'EN010003 - NI Case 3 Name' }
+				},
+				{
+					description: { contains: 'EN010003 - NI Case 3 Name' }
+				}
```

Comment:
We can probably generalise this string`'EN010003 - NI Case 3 Name'` across the file and put it in a variable

Reply:
Solved with feat(api/applications): searchString for testing (BOAS-92)